hack/make-release-artifacts.sh

```diff
@@ -75,6 +76,6 @@ if [[ ! "${git_describe}" =~ v.* ]]; then
 fi
 krew_version="${TAG_NAME:-$git_describe}"
 cp ./hack/krew.yaml ./out/krew.yaml
-sed -i "s/KREW_TAR_CHECKSUM/${tar_checksum}/g" ./out/krew.yaml
+sed -i "${checksum_sed}" ./out/krew.yaml
```

Comment:
```suggestion
sed "${checksum_sed}" ./hack/krew.yaml > ./out/krew.yaml
```
And remove the `cp ./hack/krew.yaml ./out/krew.yaml` in the previous line

Reply:
Did also integrated the next sed in it.